drgn/helpers/linux/printk.py

```diff
@@ -158,7 +158,12 @@ def add_record(current_id: int) -> None:
 
 
 def _get_printk_records_structured(prog: Program) -> List[PrintkRecord]:
-    printk_logp_type = prog.type("struct printk_log *")
+    try:
+        printk_logp_type = prog.type("struct printk_log *")
+    except LookupError:
+        # RHEL 7
```

Comment:
Could you please add a more specific reference to the kernel commit that changed this? I just added this guideline to `CONTRIBUTING` for future reference: https://github.com/osandov/drgn/blob/main/CONTRIBUTING.rst#linux-kernel-helpers

```suggestion
        # Before Linux kernel commit 62e32ac3505a ("printk: rename struct log
        # to struct printk_log") (in v3.11), records were "struct log" instead
        # of "struct printk_log".
```

Reply:
Sure, thanks for checking this. I've just pushed the change.